.travis.yml

```diff
@@ -2,5 +2,4 @@ language: clojure
 lein: lein2
 script: lein2 test-all
 jdk:
-  - openjdk7
-  - openjdk6
+  - oraclejdk8
```

Comment:
Ah, I see. That makes sense now.

I think though we still want to test with JDK 7. I don't mind if we have testing dependencies that are pulled in automatically, I just don't want the user to have to install things manually (like phantom). Sorry if I wasn't clear about that!